README.md

```diff
@@ -61,6 +67,76 @@ make migrate
 make collectstatic
 ```
 
+### 2. Run the server
+#### A. From PyCharm Professional
+```
+# go to deployment/ansible/development/group_vars
+cd deployment/ansible/development/group_vars
+cp all.sample.yml all.yml
+```
+
+- edit line 6 (*remote_user*), 8 (*remote_group*), and 10 (*project_path*) in all.yml accordingly
+  - make sure that *remote_user* align with your local user
+  - *remote_group* is likely not change if using linux and macOS
+  - *project_path* is equal to with your cloned directory
+
+```
+# go to deployment/ansible
+mkdir tmp
+make setup-ansible
```

Comment:
it fails here.
after `mkdir` we are still in `ford3/deployment/ansible`
`cd ..`
`make setup-ansible`